examples/stable-diffusion/README.md

```diff
@@ -235,6 +235,39 @@ python ../gaudi_spawn.py \
     --bf16 \
     --distributed
 ```
+
+Here is how to generate SDXL images with optimized pipeline:
+```bash
+python text_to_image_generation.py \
+    --model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 \
+    --prompts "Sailing ship painting by Van Gogh" \
+    --num_images_per_prompt 28 \
+    --batch_size 7 \
+    --image_save_dir /tmp/stable_diffusion_xl_images \
+    --scheduler euler_discrete \
+    --use_habana \
+    --use_hpu_graphs \
+    --gaudi_config Habana/stable-diffusion \
+    --bf16 \
+    --optimize
+```
+
+Here is how to generate SDXL images with optimized pipeline in fp8:
```

Comment:
Where is instruction for the measurement?

Reply:
We are using pre generated measurement from other sdxl repo from model garden which has dataset support to generate the measurement. OH sdxl repo runs only inference and doesn't have dataset and support yet to generate the measurement.